src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -134,7 +134,51 @@ public void SendKeys_Default_IsSupported()
 
             element.SendKeys("Hello World!");
 
-            Assert.That(element.Text, Is.EqualTo("Hello World!"));
+            Assert.That(element.Text, Is.EqualTo("Hello World!Test TextBox"));
+        }
+
+        [Test]
+        public void SendKeys_ShiftedCharacter_ShiftIs_Released()
```

Comment:
```suggestion
        public void SendKeys_ShiftedCharacter_ShiftIsReleased()
```

Reply:
Apologies, one day I'll get the hang of this convention ;)